products/BellHybrid/apps/application-bell-settings/models/alarm_settings/SnoozeListItemProvider.cpp

```diff
@@ -49,30 +48,30 @@ namespace app::bell_settings
     }
 
     SnoozeListItemProvider::SnoozeListItemProvider(AbstractSnoozeSettingsModel &model,
```

Comment:
I'd rename this value so that it's more meaningful, especially when two different models are passed to this object (please update the name in declaration in `.hpp` file too):

```suggestion
    SnoozeListItemProvider::SnoozeListItemProvider(AbstractSnoozeSettingsModel &settingsModel,
```